test/checks/shared/has-visible-text.js

```diff
@@ -52,12 +52,28 @@ describe('has-visible-text', function() {
 			);
 		});
 
-		it('should return undefined if element is named from contents', function() {
+		it('should return false if there is no visible text', function() {
```

Comment:
As above, this is a false positive. I'm OK with a tech debt ticket if this is too difficult to solve in one PR.